OpenMTR/Models/ProgressBar.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+
+namespace OpenMTR
+{
+    /// <summary>
+    /// An ASCII progress bar
+    /// https://gist.github.com/DanielSWolf/0ab6a96899cc5377bf54
+    /// Console progress bar. Code is under the MIT License: http://opensource.org/licenses/MIT
+    /// </summary>
+    public class ProgressBar : IDisposable, IProgress<double>
+    {
+        private const int blockCount = 10;
+        private readonly TimeSpan animationInterval = TimeSpan.FromSeconds(1.0 / 8);
+        private const string animation = @"|/-\";
+
+        private readonly Timer timer;
+
+        private double currentProgress = 0;
+        private string currentText = string.Empty;
+        private bool disposed = false;
+        private int animationIndex = 0;
+        private string _title = "";
```

Comment:
This private variable name is formed correctly but the others do not follow convention. IE : _animationIndex